recipes/hyperscan/all/conanfile.py

```diff
@@ -1,9 +1,13 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, rename, get, copy, rmdir
+from conan.tools.scm import Version
+from conan.tools.build import check_min_cppstd
 
 import os
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.53.0"
```

Comment:
Any motivation to require 1.53.0 here?

Reply:
I don't know what version is needed by the rewrite I did for conan V2. I just tested with 1.53.0.
You have an advice ?